README.md

```diff
@@ -0,0 +1,47 @@
+Create RPMs from a conda-gitenv
+===============================
+
+conda-rpms is designed to convert a resolved [conda-gitenv](https://github.com/SciTools/conda-gitenv) environment into a collection of RPM specs suitable for deployment on compatible platforms (Red Hat Enterprise Linux, Fedora, etc.).
+
+A high level description of the motivation for the creation of RPMs from a [conda-gitenv](https://github.com/SciTools/conda-gitenv) is described in the [centrally managed scientific software workflow](https://rawgit.com/pelson/conda-rpms/master/tmp_overview_docs/scitools-env-workflow.html) document.
+
+The code to do the conda distribution to RPM spec transformation itself is relatively simple, drawing on tools such as gitpython, Jinja2 and conda.
+
+Because a built RPM knows its destined installation location, a number of RPM abstractions have been made that enable us to retain conda's hard-linking and relocatability benefits.
+
+RPM Types
+=========
+
+Package RPM
+-----------
+
+Named in the form ``<RPM prefix>-pkg-<pkg name>-<pkg version>-<pkg build id>`` a package RPM represents the conda "package cache" (the thing that normally lives in `<root prefix>/.pkgs/<pkg name>-<pkg version>-<pkg build id>`). A package RPM *does not* express its dependencies and can not be usefully installed as a standalone entity.
```

Comment:
I'm not sure about 'RPM name format' 

Maybe some special formatting?
**RPM name format:**`<RPM prefix>-pkg-<pkg name>-<pkg version>-<pkg build id> `

I just want it to be clear that that section isn't a part of the explanation of what a package rpm is.